python/add_corona.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+
+#===================================================
+#
+#    Copyright (c) 2025
+#      SMASH Hybrid Team
+#
+#    GNU General Public License (GPLv3 or later)
+#
+#===================================================
+
+import argparse
+import numpy as np
+import sys
+import os
+import subprocess
+
+'''
+    This script adds the corona particles from the initial SMASH run
```

Comment:
Naively, this sounds to me like there's only one list from the IC stage. Maybe add that this includes a list of particles during the Hydro stage that didn't undergo fluidization due to the live time of the fluid (if I understood this right). But this might be already to verbose, so just a suggestion that you can use or disregard at will.

Reply:
i put generically "and possibly others" as there is no specific mention of the other corona coming from hydro (nor it needs to be enforced)

Reply:
Just a suggestion, but why not more verbose and precise like:


```suggestion
    This script adds the corona particles from the IC and possibly
    the Hydro stage to the sampled particles list.
```